integration/docker_compose_test.go

```diff
@@ -0,0 +1,80 @@
+package integration
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/containous/traefik/integration/try"
+	"github.com/containous/traefik/testhelpers"
+	"github.com/containous/traefik/types"
+	"github.com/go-check/check"
+	"github.com/libkermit/docker"
+	checker "github.com/vdemeester/shakers"
+)
+
+const (
+	composeProject = "minimal"
+)
+
+// Docker test suites
+type DockerComposeSuite struct {
+	BaseSuite
+	project *docker.Project
+}
+
+func (s *DockerComposeSuite) SetUpSuite(c *check.C) {
+	s.createComposeProject(c, composeProject)
+	s.composeProject.Start(c)
+}
+
+func (s *DockerComposeSuite) TearDownSuite(c *check.C) {
+	// shutdown and delete compose project
+	if s.composeProject != nil {
+		s.composeProject.Stop(c)
+	}
+}
+
+func (s *DockerComposeSuite) TestComposeScale(c *check.C) {
+	var serviceCount = 2
```

Comment:
not needed, only one occurrence.

Reply:
serviceCount is used [L44](https://github.com/containous/traefik/pull/3190/files/0c9ca7ec5462f6c9b98e9e4dfdba7d37dce762a6#diff-42a4390e47bc8cbbb2db8a2ce35202b5R44) and [L76](https://github.com/containous/traefik/pull/3190/files/31858a49d221493f5963b6a6f7a6f33f9430f06f#diff-42a4390e47bc8cbbb2db8a2ce35202b5R76)